docs/reference/values.md

```diff
@@ -10,11 +10,12 @@ You can read and write [HTML data attributes](https://developer.mozilla.org/en-U
 <meta data-controller="callout" data-callout-text-value="data-loader-url-value=&quot;/messages&quot;">
 
 ```html
-<div data-controller="loader"
-     data-loader-url-value="/messages">
+<div data-controller="loader" data-loader-url-value="/messages">
 </div>
 ```
 
+The definition of the value data attribute should be specified in the same location as the controller data attribute.
```

Comment:
```suggestion
Data attributes used for values must be declared on the controlled element, the same one that has the `data-controller `attribute.
```

Just an idea, I think `must` is better than `should` here, also the concept of 'same location' could be confusing.

Reply:
Great suggestions. Thanks!